libs/sdk-ui-dashboard/src/presentation/widget/common/configuration/AttributeFilterConfiguration.tsx

```diff
@@ -20,20 +29,58 @@ export const AttributeFilterConfiguration: React.FC<IAttributeFilterConfiguratio
     const dfMap = useDashboardSelector(selectAttributeFilterDisplayFormsMap);
     const attrMap = useDashboardSelector(selectAllCatalogAttributesMap);
 
+    const displayForms = useMemo(() => {
```

Comment:
Separate this logic not hook useAttributeFilterConfiguration

Reply:
Moved to hook.